src/logger.js

```diff
@@ -0,0 +1,52 @@
+function wikiaJWPlayerLogger(options) {
+	var servicesDomain = options.servicesDomain || 'services.wikia.com',
+		loggerPath = '/event-logger/error',
+		loggerUrl = 'https://' + servicesDomain + loggerPath,
+		prefix = 'JWPlayer';
```

Comment:
if this has a prefix - should all logs have jwplayer word in them? sounds like noise